hw/top_earlgrey/data/ip/chip_uart_testplan.hjson

```diff
@@ -93,6 +93,7 @@
       lc_states: ["PROD"]
       features: ["UART.PARITY"]
       tests: []
+	  bazel: ["//sw/device/tests:uart_parity_test"]
```

Comment:
The indentation seems wrong.

Reply:
Thanks, I'd added a literal tab character somehow.